ttnn/cpp/ttnn/operations/sliding_window/halo/device/untilize_with_halo_v2_program_factory.cpp

```diff
@@ -364,6 +367,22 @@ operation::ProgramWithCallbacks inplace_untilize_with_halo_multi_core_v2(
         create_circular_buffer(program, all_cores, cb_indices.src_cb_id, in_df, input_npages, in_page_size, src_buffer);
 
     uint32_t input_to_writer_cb_id = cb_indices.src_cb_id;
+    if (!skip_untilize) {
+        cb_indices.untilize_out_cb_id = cb_indices.get_next_cb_id();
+        input_to_writer_cb_id = cb_indices.untilize_out_cb_id;
+        // output of untilize from compute kernel goes into this CB
+        uint32_t output_ntiles = ntiles_per_block * input_nblocks_per_core;
```

Comment:
This will defeat the purpose of inplace halo, as you will allocate here aditional CB whichi the same size or bigger than the input.
Idea should be that we want untialize to be inplace as well.